examples/js/loaders/OBJLoader.js

```diff
@@ -260,72 +267,96 @@ THREE.OBJLoader = ( function () {
 
 			},
 
-			addFace: function ( a, b, c, d, ua, ub, uc, ud, na, nb, nc, nd ) {
+			addFace: function ( args ) {
 
 				var vLen = this.vertices.length;
+				var uvLen = this.uvs.length;
+				var nLen = this.normals.length;
 
-				var ia = this.parseVertexIndex( a, vLen );
-				var ib = this.parseVertexIndex( b, vLen );
-				var ic = this.parseVertexIndex( c, vLen );
-				var id;
+				var addingFaceMode = args.pop();
+				var ivs = [ ], ius = [ ], ins = [ ];
 
-				if ( d === undefined ) {
+				if( addingFaceMode === adding_face_modes.vertex ) {
```

Comment:
Also, you can use http://zz85.github.io/mrdoobapproves/ to check your formatting.